docs/src/appendix/cast/multicall.md

```diff
@@ -27,4 +34,18 @@ call_type = "invoke"
 contract_address = "map_contract"
 function = "put"
 inputs = ["0x123", "234"]
-```
\ No newline at end of file
+```
+
+## `new`
+
+Generates an empty template for the multicall `.toml` file that may be later use with the `run` subcommand.
+
+### `--output_path, -p <PATH>`
+Optional.
+
+When provided, it specifies where the template should be saved. If omitted, the template contents are going to be printed out to the stdout.
+
+### `--overwrite, -o <OVERWRITE>`
+Optional.
+
+If the `--output-path` argument is provided and such a file already exists, this flag decides whether it should be overwritten or not.
```

Comment:
```suggestion
Overwrites the file specified by `--output-path`, if it exists.
```

Reply:
I think the original sentence's better, in the suggested one, in the latter part, "it" is confusing - it is not clear what it refers to (either it's the file or the output path).

Besides, I think the original sentence is easier to understand.